src/SFA.DAS.FAA.Domain/Applications/GetSubmittedApplications/GetSubmittedApplicationsApiResponse.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+
+namespace SFA.DAS.FAA.Domain.Applications.GetSubmittedApplications
```

Comment:
i think this needs to exist in the Users folder and relate more to something like
```
GetAccountDeletionApplicationsToWithdrawApiResponse
```

Reply:
Moved and renamed as appropriate